model/jpa/pom.xml

```diff
@@ -84,7 +84,50 @@
                     <groupId>com.h2database</groupId>
                     <artifactId>h2</artifactId>
```

Comment:
Do we need `h2` ? 

Or, even better - we can execute the tests using both: h2 and derby/javaDB